experimental/addons/koordinator/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+name: koordinator
+version: 1.0.0
```

Comment:
Should this be `1.1.1`, which is the version of the Koordinator helm chart to install?

Reply:
Done 👍🏽